config/readinglist.ini

```diff
@@ -15,6 +15,7 @@ fxa-oauth.client_id = 89513028159972bc
 fxa-oauth.client_secret = 9aced230585cc0aa2932e2eb871c9a3a7d6458e59ccf57eb610ea0a3467dd800
 fxa-oauth.oauth_uri = https://oauth-stable.dev.lcip.org
 fxa-oauth.scope = profile
+fxa-oauth.cache_verification = 300
```

Comment:
Nit: it's often considered good practice to name the setting using the type of unit used; here, `cache_ttl_seconds` would be meaningful.